packages/server-wallet/src/__test-with-peers__/utils.ts

```diff
@@ -0,0 +1,46 @@
+import {ChannelResult} from '@statechannels/client-api-schema';
+
+import {Wallet} from '..';
+import {TestMessageService} from '../message-service/test-message-service';
+import {MessageHandler, MessageServiceInterface} from '../message-service/types';
+
+// TODO: Is there a cleaner way of writing this?
+export async function expectLatestStateToMatch(
```

Comment:
@andrewgordstewart you've written a lot of test helpers, is there a nicer way of doing this?

Reply:
What don't you find nice about this?

Reply:
The main issue is that you have to `await` `expectLatestStateToMatch` which could be easy to miss when writing a test. Probably not worth worrying too much about it.

Reply:
I don't think there's any way around that, right?

Reply:
I thought there might be a more elegant way using a custom jest matcher? Either way, I don't think it's worth the time.